certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -930,93 +396,80 @@
         "responses": {
-          "default": {
+          "200": {
             "description": "default",
-            "schema": {}
+            "schema": {
+              "$ref": "#/definitions/AddressSaveData"
+            }
           }
         },
-        "summary": "Get Committee Members by committee ID or code",
-        "description": "Get Committee Members by Committee ID or Code",
-        "operationId": "Get-Committee-Members-by-committee-ID-or-code",
+        "summary": "Add or Update Address to Organization",
+        "description": "Add or Update Address to Organization",
+        "operationId": "Add-or-Update-Address-to-Organization",
         "parameters": [
           {
-            "name": "ID Or Code",
+            "name": "ID or Record Number",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "Committee ID or Code",
-            "x-ms-summary": "Committee ID or Code",
-            "x-ms-url-encoding": "single"
+            "description": "Organization ID or Record Number.",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Organization ID or Record Number."
           },
           {
-            "name": "Page Number",
-            "in": "path",
+            "name": "Content-Type",
+            "in": "header",
             "required": true,
-            "type": "integer",
-            "default": 1,
-            "description": "Page Number",
-            "x-ms-summary": "Page Number",
-            "x-ms-visibility": "advanced",
-            "x-ms-url-encoding": "single"
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
           },
           {
-            "name": "Term",
-            "in": "query",
+            "name": "body",
+            "in": "body",
             "required": false,
-            "type": "integer",
-            "description": "Term",
-            "x-ms-summary": "Term"
-          },
-          {
-            "name": "Position Codes",
-            "in": "query",
-            "required": false,
-            "type": "string",
-            "description": "Position Codes",
-            "x-ms-summary": "Position Codes"
-          },
-          {
-            "name": "Content-Type",
-            "in": "header",
-            "required": true,
-            "type": "string",
-            "default": "application/json",
-            "x-ms-visibility": "internal"
+            "schema": {
+              "$ref": "#/definitions/AddressSaveData"
+            }
           }
         ]
       }
     },
-    "/api/v1/Individuals/{id}/Activities": {
+    "/api/v1/Individuals/{ID or Record Number}/CustomFieldsList": {
       "post": {
-        "tags": ["Individuals"],
+        "tags": [
+          "Individuals"
+        ],
         "responses": {
           "200": {
-            "description": "Success",
+            "description": "default",
             "schema": {}
```

Comment:
Please check the response definition at http://abc.mpxdemo.com/api/help/Api/POST-api-v1-Individuals-idOrRecordNumber-CustomFieldsList

Reply:
When I add the schema, it causes the flow to fail and result in the following. However, I remove the schema and it works.
![Screenshot 2021-11-25 185937](https://user-images.githubusercontent.com/83984082/143512049-e2d31fba-6f58-43fe-b6ce-a2ccb5657c80.png)
![Screenshot 2021-11-25 185919](https://user-images.githubusercontent.com/83984082/143512050-ac251f98-91c7-427e-a86b-4913f609555a.png)

Reply:
There is an error with the swagger definition, the response is an array of `CustomFieldResultData` http://abc.mpxdemo.com/api/help/ResourceModel?modelName=CustomFieldResultData

Reply:
![Screenshot 2021-11-28 205754](https://user-images.githubusercontent.com/83984082/143801897-487cd987-b0c6-46c5-90e2-32a783518023.png)
Adding the response definition causes paconn to run an error with it. In addition, the flow doesnt work. I removed the response definition and paconn succeeded and the flow works.

Reply:
I don't get any errors adding it like this, can you check it again?
```
"responses": {
  "200": {
    "description": "default",
    "schema": {
      "type": "array",
      "items":{
        "$ref": "#/definitions/CustomFieldResultData"
      }
    }
  }
},
```

Reply:
I got the flow working, but I am still gettting the paconn error. 
![Screenshot 2021-11-30 145854](https://user-images.githubusercontent.com/83984082/144127127-4404fb3e-3eca-4fdb-bec4-ed3d039acaae.png)
You arent getting the error when run through paconn? What does your definition for customfieldresultdata look like?

Reply:
I didn't make any changes to the CustomFieldResultData definition:
```
"CustomFieldResultData": {
  "type": "object",
  "properties": {
    "status": {
      "type": "string",
      "description": "Status.",
      "title": "Status"
    },
    "message": {
      "type": "string",
      "description": "Message.",
      "title": "Message"
    },
    "customFieldName": {
      "type": "string",
      "description": "Custom Field Name.",
      "title": "Custom Field Name"
    }
  }
},
```

Reply:
My paconn still doesnt validate it. Any suggestions?

Reply:
Can you commit the changes so I can check?

Reply:
Im not sure whats different but, I checked paconn again with the file I committed to this PR and it validated this time.